README.md

```diff
@@ -1,2 +1,6 @@
-# code-review
-This repo is a part of learning how to get code review 
+# This page is a part of Microverse trial as learning purpose.
+# create survey form.
+# Who is behind Microverse?
+We are a fully remote and global company, with team members across 20+ countries,<br> who come together to make an impact through education.<br>
+Get to know the global team 
```

Comment:
You have done a great job making use of inline comments, you have also done a great job adding a reason for requesting the change👍😉 However, I think we can make some improvements to your comments by improving our mode of communication😉 rememver, we were taught in the [previous lesson](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/give_code_review_basics.md) on "how to give a code review" to be suggestive when giving reviews rather than giving orders😉

<img width="866" alt="no_orders_cr" src="https://user-images.githubusercontent.com/64643995/213978802-42d7d201-3abe-4aa6-a010-0a166d28a8ec.png">

we can make improvements to your statement for instance "I think your readme file can have more description  because it is important to talk about your repo's summary for viewers' attention😉." looks more suggestive and polite😉 you can go through the [previous lesson](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/give_code_review_basics.md) for more details and make improvements😉